components/style/properties/gecko.mako.rs

```diff
@@ -1213,6 +1213,21 @@ fn static_assert() {
 
     <%call expr="impl_coord_copy('line_height', 'mLineHeight')"></%call>
 
+    pub fn set_word_spacing(&mut self, v: longhands::word_spacing::computed_value::T) {
+        use values::computed::LengthOrPercentage::*;
+
+        match v.0 {
+            Some(lop) => match lop {
+                Length(au) => self.gecko.mWordSpacing.set_value(CoordDataValue::Integer(au.0)),
+                Percentage(f) => self.gecko.mWordSpacing.set_value(CoordDataValue::Percent(f)),
+                Calc(l_p) => self.gecko.mWordSpacing.set_value(CoordDataValue::Calc(l_p.into())),
+            },
+            None => self.gecko.mWordSpacing.set_value(CoordDataValue::Auto),
```

Comment:
None should be zero I think. Or None. The spec says 0 but we might be able to get away with using None if gecko layout treats it as the same.